packages/rmb_direct_client/DEVELOPERS.README.md

```diff
@@ -0,0 +1,23 @@
+# Types Update Guide for Developers
+
+## Introduction
+
+This guide outlines the steps to update the types used in the `RMB direct` Client to reflect changes in messages sent over the RMB.
+
+### Steps
+
+1. **Update types.proto File**: Update the [types.proto](lib/types.proto) file with any changes to the message structure.
+
+2. **Install Protobuf v21.2**: Install Protobuf version [v21.2](https://github.com/protocolbuffers/protobuf/releases/tag/v21.2) on your system.
```

Comment:
might be best to replace v21.2 with the ```version in the package.json``` or with the ```version that was used to create the imported types.proto``` as the version may change in the future